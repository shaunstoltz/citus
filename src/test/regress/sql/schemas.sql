SELECT nspname FROM pg_namespace WHERE nspname LIKE 'test_schema%' ORDER BY nspname;
SELECT * FROM test_schema_5.test_table ORDER BY a;
