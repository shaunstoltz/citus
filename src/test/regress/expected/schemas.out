SELECT nspname FROM pg_namespace WHERE nspname LIKE 'test_schema%' ORDER BY nspname;
    nspname
---------------------------------------------------------------------
 test_schema_1
 test_schema_3_renamed
 test_schema_5
(3 rows)

SELECT * FROM test_schema_5.test_table ORDER BY a;
 a
---------------------------------------------------------------------
 1
 2
(2 rows)

